libs/features/mas/src/checkout-mixin.js

```diff
@@ -1,18 +1,18 @@
-import { ignore } from './external.js';
+import { ignore } from '@dexter/tacocat-core';
```

Comment:
I suggest to remove this import..
ignore function looks like this:
```
const ignore = () => {
    /* do nothing */
};
```

imo no point to import it from other package, maybe we could remove dependency on tacocat-core at some moment

Reply:
or if you see value in ignore function i'd redefine it in utilities.. but its so simple it could be just defined in this file on the fly as well

Reply:
I'll remove it